Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/FixedProjectiveConstraint.cpp

```diff
@@ -89,8 +89,8 @@ void FixedProjectiveConstraint<Rigid3Types>::draw(const core::visual::VisualPara
         }
         else
         {
-            for (SetIndex::const_iterator it = indices.begin(); it != indices.end(); ++it)
-                points.push_back(x[*it].getCenter());
+            for (const unsigned int indice : indices)
+                points.push_back(x[indice].getCenter());
```

Comment:
```suggestion
            for (const unsigned int index: indices)
                points.push_back(x[index].getCenter());
```

I guess clang-tidy does not know that the singular of indices is index 😅